### ISSUE TITLE
2.1.1 - Core requirement: Images detectable - Test needs update

### ISSUE THREAD

**Issue by JasonAment** [w3c/wcag3#606]
Combining two test modalities "code inspections" and "screen reader inspections" into a single test case opens it up to incorrect interpretation. 

If these are equivalent tests under different conditions based on using alternate modalities to test the same scenario, they should be split into two tests with alternate conditions with number ordering at the same level:

Sample suggestion:

For each non-decorative image that can be detected with a code inspector:
1. Check the code to see if it has been marked up in a way that makes it detectable.

Alternatively, for each non-decorative image that cannot be programmatically determined through code inspection:
1. For technologies where the code cannot be checked, use a screen reader to test that the image is detectable.
